src/base/bittorrent/session.cpp

```diff
@@ -5250,7 +5249,7 @@ void Session::processTrackerStatuses()
         const std::vector<lt::announce_entry> trackerList = torrent->nativeHandle().trackers();
         for (const lt::announce_entry &announceEntry : trackerList)
         {
-            const auto trackerURL = QByteArray::fromRawData(announceEntry.url.c_str(), announceEntry.url.size());
+            const auto trackerURL = QByteArray::fromRawData(announceEntry.url.c_str(), static_cast<int>(announceEntry.url.size()));
```

Comment:
The parameter type in Qt6 is `qsizetype`.

Reply:
Change dropped. Also addressed other comments.